modules/gapi/test/gapi_frame_tests.cpp

```diff
@@ -17,15 +17,13 @@ G_API_OP(GBlurFrame, <GMat(GFrame)>, "test.blur_frame") {
     }
 };
 
-GAPI_OCV_KERNEL(OCVBlurFrame, GBlurFrame)
```

Comment:
Here and below. 
Why did you change the brackets ?

Reply:
Consistency reasons - look at this file